sw/airborne/modules/imu/imu_mpu9250_i2c.c

```diff
@@ -57,6 +57,7 @@ PRINT_CONFIG_MSG("Gyro/Accel output rate is 2kHz at 8kHz internal sampling")
 #define IMU_MPU9250_GYRO_LOWPASS_FILTER MPU9250_DLPF_GYRO_41HZ
 #define IMU_MPU9250_ACCEL_LOWPASS_FILTER MPU9250_DLPF_ACCEL_41HZ
 PRINT_CONFIG_MSG("Gyro/Accel output rate is 100Hz at 1kHz internal sampling")
+#error Non-default PERIODIC_FREQUENCY: please define IMU_MPU9250_GYRO_LOWPASS_FILTER, IMU_MPU9250_ACCEL_LOWPASS_FILTER and IMU_MPU9250_SMPLRT_DIV.
```

Comment:
If it is an error, is it even necessary to define filter values ?

Reply:
I think I noticed an #error message doesn't stop you from finishing the compilation, so like this you could notice during compilation but you can also ignore it and flash... Not sure what is the desired behavior here, I would be fine with either.

Reply:
#error stops the compilation. Use a #warning instead if you want to still compile.